docs/source/guides.md

```diff
@@ -0,0 +1,6 @@
+# Text Generation Inference Guides
+
+Text Generation Inference (TGI) is integrated into the Hugging Face ecosystem and can be used in many different ways. Below is a list of guides to help you get started with TGI.
```

Comment:
Maybe this is better to include in either index or README instead of a dedicated section but feel free to dismiss if you disagree